_unittest/test_12_1_PostProcessing.py

```diff
@@ -102,7 +102,17 @@ def test_01B_Field_Plot(self):
         plot1.update()
         assert self.aedtapp.post.field_plots[plot1.name].IsoVal == "Tone"
         assert plot1.change_plot_scale(min_value, "30000", scale_levels=50)
-        assert self.aedtapp.post.create_fieldplot_volume("inner", "Vector_E", setup_name, intrinsic)
+        assert not self.aedtapp.post.create_fieldplot_volume("inner", "Vector_E", setup_name, intrinsic)
+        inner = self.aedtapp.modeler.objects_by_name["inner"]
+        inner.solve_inside = True
+        inner_plot = self.aedtapp.post.create_fieldplot_volume("inner", "Vector_E", setup_name, intrinsic)
+        self.aedtapp.analyze(self.aedtapp.active_setup)
+        assert inner_plot
```

Comment:
Shouldn't that call me moved before analyze ? It makes me think that analyze is required for the assert to work. That's not the case right ?